packages/host/app/components/login.gts

```diff
@@ -0,0 +1,120 @@
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+import { action } from '@ember/object';
+import { on } from '@ember/modifier';
+import { tracked } from '@glimmer/tracking';
+import { restartableTask } from 'ember-concurrency';
+import {
+  BoxelHeader,
+  BoxelInput,
+  Button,
+  FieldContainer,
+  LoadingIndicator,
+} from '@cardstack/boxel-ui';
+import { isMatrixError } from '../lib/matrix-utils';
+import type MatrixService from '../services/matrix-service';
+import { type IAuthData } from 'matrix-js-sdk';
+
+const TRUE = true;
```

Comment:
How come a constant is needed for `true`?

Reply:
The prettier for glint turns a `false` in the tea plate to `”false”`—very annoying.